src/qml/ProjectCreationScreen.qml

```diff
@@ -0,0 +1,346 @@
+        }
+      }
+
+      QfExpandableGroupBox {
+        id: takeNotesGroupBox
+        title: qsTr("Take notes?")
+        width: parent.width
+        checked: true
+        content: Column {
+          id: takeNotesColumn
+          anchors.left: parent.left
+          anchors.right: parent.right
+
+          Label {
+            text: qsTr("Quickly capture notes with date, time, and comments. Optionally, attach images to enrich your notes.")
+            font: Theme.defaultFont
+            color: Theme.secondaryTextColor
+            wrapMode: Text.WordWrap
+            width: parent.width
+          }
+
+          CheckBox {
+            id: takeMediaCheckBox
+            text: qsTr("Take image and video attachments")
+            font: Theme.defaultFont
+            indicator.height: 16
+            indicator.width: 16
+            indicator.implicitHeight: 24
+            indicator.implicitWidth: 24
+            leftPadding: 1
+            checked: true
+          }
+        }
+      }
+
+      QfExpandableGroupBox {
+        id: trackPositionGroupBox
+        title: qsTr("Track your position?")
+        width: parent.width
+        checked: true
+        content: Column {
+          id: trackPositionColumn
+          anchors.left: parent.left
+          anchors.right: parent.right
+
+          Label {
+            text: qsTr("Record your location every second, along with date and time, to keep a precise track of your movements.")
+            font: Theme.defaultFont
+            color: Theme.secondaryTextColor
+            wrapMode: Text.WordWrap
+            width: parent.width
+          }
+
+          CheckBox {
+            id: autoTrackPositionCheckBox
+            text: qsTr("Begin tracking automatically")
+            font: Theme.defaultFont
+            indicator.height: 16
+            indicator.width: 16
+            indicator.implicitHeight: 24
+            indicator.implicitWidth: 24
+            leftPadding: 1
+            checked: true
+          }
+        }
+      }
+
+      QfExpandableGroupBox {
+        id: qfieldCloudGroupBox
+        title: qsTr("Backup & collaborate?")
+        width: parent.width
+        checked: true
+        icon: "qrc:/themes/qfield/nodpi/ic_cloud_active_24dp.svg"
+
+        content: Column {
+          id: databaseAndColabrationColumn
+          width: parent.width
+          spacing: 0
+
+          Label {
+            text: qsTr("QFieldCloud allows to synchronize and merge the data collected by your team in QField. From small individual projects to large data collection campaigns. ") + "<a href=\"https://qfield.cloud/\">" + qsTr("Learn more about QFieldCloud here") + "</a>."
+            font: Theme.defaultFont
+            color: Theme.secondaryTextColor
+            wrapMode: Text.WordWrap
+            width: parent.width
+            textFormat: Text.RichText
+            onLinkActivated: link => Qt.openUrlExternally(link)
+          }
+        }
+      }
+    }
+  }
+
+  QfButton {
```

Comment:
Here we'll want a floating Rectangle {} over the configuration layout with a semi opaque background, within which this single QfButton will exist.